uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/IOServiceIndexedTest.java

```diff
@@ -95,7 +97,7 @@ public void setup() throws URISyntaxException {
         when(mockProvider.newFileSystem(any(Path.class), any())).thenReturn(mock(MockFileSystem.class, RETURNS_DEEP_STUBS));
         when(mockProvider.getFileSystem(any(URI.class))).thenReturn(mock(MockFileSystem.class, RETURNS_DEEP_STUBS));
 
-        ioService = spy(new IOServiceIndexedImpl(indexEngine, executorService, indexersFactory, dispatcherFactory));
+        ioService = spy(new IOServiceIndexedImpl(indexEngine, DefaultApplicationScopedProducerutorService, fsWatchExecutorService, indexersFactory, dispatcherFactory));
```

Comment:
This caused the FDB failed.
```suggestion
        ioService = spy(new IOServiceIndexedImpl(indexEngine, indexingExecutorService, fsWatchExecutorService, indexersFactory, dispatcherFactory));
```